python/src/scipp/compat/mantid.py

```diff
@@ -238,19 +243,13 @@ def validate_and_get_unit(unit, allow_empty=False):
             'Q^2',
             sc.units.dimensionless / (sc.units.angstrom * sc.units.angstrom)
         ],
-        "Label": ['spectrum', sc.units.dimensionless],
+        "Spectrum": ['spectrum', sc.units.dimensionless],
         "Empty": ['empty', sc.units.dimensionless],
         "Counts": ['counts', sc.units.counts]
     }
 
     if unit not in known_units.keys():
-        if allow_empty:
-            return ['unknown', sc.units.dimensionless]
-        else:
-            raise RuntimeError("Unit not currently supported."
-                               "Possible values are: {}, "
-                               "got '{}'. ".format(
-                                   [k for k in known_units.keys()], unit))
+        return [str(unit), sc.units.dimensionless]
```

Comment:
Yes, this is most likely better. Previous behaviour probably annoying in most cases.